### PR TITLE
chore: add new dcos-ui package version

### PR DESCRIPTION
**Pull request by juliangieseke**
This updates ui packages devliered for ui-update-service to version that we shipped with 1.13.2 for customers running 1.13.1 or 1.13.0.

### CODE REVIEW

File: repo/packages/D/dcos-ui/2/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "4.0",
+  "name": "dcos-ui",
+  "version": "2.82.6+0e5782c0",
```
Comment (TattdCodeMonkey) [inline repo/packages/D/dcos-ui/2/package.json]: we don't want a sha in this version.

**Review by TattdCodeMonkey — CHANGES_REQUESTED** [review]

File: repo/packages/D/dcos-ui/2/package.json (hunk 1 above)
Comment (TattdCodeMonkey) [reply repo/packages/D/dcos-ui/2/package.json]: at least we should test this.

Comment (juliangieseke) [reply repo/packages/D/dcos-ui/2/package.json]: CI also complains, yeah… not sure why tho, its valid semver 🤔

**Review by TattdCodeMonkey — APPROVED** [review]
Tested this with a stub universe, DC/OS 1.13.1 and `dcos-ui-update-service` and everything was 💯

**Pull request merged**